.github/workflows/ci.yaml

```diff
@@ -57,26 +57,87 @@ jobs:
         with:
           version: 1.2.1
 
-      - name: Lint & test aserto package
+      - name: Run lint
         run: |
-          cd packages/aserto
           poetry install
           poetry run pyright .
+      
+      - name: Run tests
+        run: |
           poetry run pytest -vv
-          cd ../..
+  release:
+    runs-on: ubuntu-latest
+    needs: test
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags/v')
 
-      - name: Lint & test aserto-idp package
-        run : |
-          cd packages/aserto-idp
-          poetry install
-          poetry run pyright .
-          poetry run pytest -vv
-          cd ../..
+    name: Release to pypi
+    steps:
+      - name: Read Configuration
+        uses: hashicorp/vault-action@v2.4.1
+        id: vault
+        with:
+          url: ${{ env.VAULT_ADDR }}
+          token: ${{ secrets.VAULT_TOKEN }}
+          secrets: |
+            kv/data/github  "SSH_PRIVATE_KEY" | SSH_PRIVATE_KEY;
+            kv/data/pypi    "USERNAME"        | POETRY_HTTP_BASIC_PYPI_USERNAME;
```

Comment:
This no longer works. Pypi stopped allowed package publishing using username/password.
You need to use the same logic as python-directory and python-authorizer: https://github.com/aserto-dev/python-directory/blob/main/.github/workflows/ci.yaml#L117